### ISSUE TITLE
Flux-dsl does not support reduce?

### ISSUE THREAD

**Issue by erikhh** [influxdata/influxdb-client-java#83]
Noticed that the ```Flux``` class of ```flux-dsl``` does not have any ```reduce()``` methods.

Does that mean that [reduce](https://docs.influxdata.com/flux/v0.50/stdlib/built-in/transformations/aggregates/reduce/) is not currently supported by ```flux-dsl```? Or am I overlooking something?

**Comment by bednar** [comment]
Hi @erikhh,

Thanks for issue. We will fix it in [1.5.0](https://github.com/influxdata/influxdb-client-java/milestone/8) milestone.

Meanwhile you could use something like this:

```java
Flux flux = Flux.from("telegraf")
    .expression("reduce(fn: (r, accumulator) => ({sum: r._value + accumulator.sum}), identity: {sum: 0.0})")
    .expression("sum()");
```

Regards

**Comment by erikhh** [comment]
Thanks, good to know it wasn't me :-)

Looks like a solid workaround, I'll give that a go.

**Comment by bednar** [comment]
Hi @erikhh,

The issue is fixed in [1.6.0 milestone](https://github.com/influxdata/influxdb-client-java/milestone/9). 

If you would like to use a preview version use our snapshot build from repository: https://oss.sonatype.org/content/repositories/snapshots/

Regards
